From d6c31cc60714a8a90535132d127527198390b196 Mon Sep 17 00:00:00 2001
From: Jason Rogena <j.rogena@cgiar.org>
Date: Fri, 13 Jun 2014 12:53:05 +0300
Subject: [PATCH 197/213] Version 1.0.1: Added Translations

Strings added to english locale as from Version 1.0.1 added to other
locales
---
 MistroFarmer/src/main/res/values/strings_kp.xml | 5 +++++
 MistroFarmer/src/main/res/values/strings_kr.xml | 5 +++++
 MistroFarmer/src/main/res/values/strings_lu.xml | 5 +++++
 MistroFarmer/src/main/res/values/strings_nn.xml | 5 +++++
 MistroFarmer/src/main/res/values/strings_sw.xml | 5 +++++
 5 files changed, 25 insertions(+)

diff --git a/MistroFarmer/src/main/res/values/strings_kp.xml b/MistroFarmer/src/main/res/values/strings_kp.xml
index 5234b6f..0c0a5c1 100644
--- a/MistroFarmer/src/main/res/values/strings_kp.xml
+++ b/MistroFarmer/src/main/res/values/strings_kp.xml
@@ -327,4 +327,9 @@
     <string name="are_you_in_farmers_farm_kp">Are you currently in the farmer\'s farm?</string>
     <string name="accuracy_kp">Accuracy</string>
     <string name="unknown_kp">unknown</string>
+    <string name="filter_farmers_kp">Filter farmers</string>
+    <string name="all_farmers_kp">All farmers</string>
+    <string name="farmers_no_epersonnel_kp">Farmers with no extension personnel</string>
+    <string name="farmers_tied_to_kp">Farmers tied to</string>
+    <string name="super_admin_kp">Super Admin</string>
 </resources>
diff --git a/MistroFarmer/src/main/res/values/strings_kr.xml b/MistroFarmer/src/main/res/values/strings_kr.xml
index 76a31e5..3537fb8 100644
--- a/MistroFarmer/src/main/res/values/strings_kr.xml
+++ b/MistroFarmer/src/main/res/values/strings_kr.xml
@@ -326,4 +326,9 @@
     <string name="are_you_in_farmers_farm_kr">Are you currently in the farmer\'s farm?</string>
     <string name="accuracy_kr">Accuracy</string>
     <string name="unknown_kr">unknown</string>
+    <string name="filter_farmers_kr">Filter farmers</string>
+    <string name="all_farmers_kr">All farmers</string>
+    <string name="farmers_no_epersonnel_kr">Farmers with no extension personnel</string>
+    <string name="farmers_tied_to_kr">Farmers tied to</string>
+    <string name="super_admin_kr">Super Admin</string>
 </resources>
\ No newline at end of file
diff --git a/MistroFarmer/src/main/res/values/strings_lu.xml b/MistroFarmer/src/main/res/values/strings_lu.xml
index 11914a6..65959c1 100644
--- a/MistroFarmer/src/main/res/values/strings_lu.xml
+++ b/MistroFarmer/src/main/res/values/strings_lu.xml
@@ -329,4 +329,9 @@
     <string name="are_you_in_farmers_farm_lu">Are you currently in the farmer\'s farm?</string>
     <string name="accuracy_lu">Accuracy</string>
     <string name="unknown_lu">unknown</string>
+    <string name="filter_farmers_lu">Filter farmers</string>
+    <string name="all_farmers_lu">All farmers</string>
+    <string name="farmers_no_epersonnel_lu">Farmers with no extension personnel</string>
+    <string name="farmers_tied_to_lu">Farmers tied to</string>
+    <string name="super_admin_lu">Super Admin</string>
 </resources>
diff --git a/MistroFarmer/src/main/res/values/strings_nn.xml b/MistroFarmer/src/main/res/values/strings_nn.xml
index 5b47564..dbc3840 100644
--- a/MistroFarmer/src/main/res/values/strings_nn.xml
+++ b/MistroFarmer/src/main/res/values/strings_nn.xml
@@ -327,4 +327,9 @@
     <string name="are_you_in_farmers_farm_nn">Are you currently in the farmer\'s farm?</string>
     <string name="accuracy_nn">Accuracy</string>
     <string name="unknown_nn">unknown</string>
+    <string name="filter_farmers_nn">Filter farmers</string>
+    <string name="all_farmers_nn">All farmers</string>
+    <string name="farmers_no_epersonnel_nn">Farmers with no extension personnel</string>
+    <string name="farmers_tied_to_nn">Farmers tied to</string>
+    <string name="super_admin_nn">Super Admin</string>
 </resources>
diff --git a/MistroFarmer/src/main/res/values/strings_sw.xml b/MistroFarmer/src/main/res/values/strings_sw.xml
index e51d6c9..0b3c5ae 100644
--- a/MistroFarmer/src/main/res/values/strings_sw.xml
+++ b/MistroFarmer/src/main/res/values/strings_sw.xml
@@ -327,4 +327,9 @@
     <string name="are_you_in_farmers_farm_sw">Are you currently in the farmer\'s farm?</string>
     <string name="accuracy_sw">Accuracy</string>
     <string name="unknown_sw">unknown</string>
+    <string name="filter_farmers_sw">Filter farmers</string>
+    <string name="all_farmers_sw">All farmers</string>
+    <string name="farmers_no_epersonnel_sw">Farmers with no extension personnel</string>
+    <string name="farmers_tied_to_sw">Farmers tied to</string>
+    <string name="super_admin_sw">Super Admin</string>
 </resources>
-- 
2.7.1

